systems/notification/mailer/pkg/db/model.go

```diff
@@ -0,0 +1,19 @@
+package db
+
+import (
+	"time"
+
+	uuid "github.com/ukama/ukama/systems/common/uuid"
+)
+
+type Mailing struct {
+	MailId    uuid.UUID  `gorm:"primaryKey;type:uuid"`
```

Comment:
Should this mailId be sent in email also for reciever so that he can refer to Id in further communication if required ?

Reply:
It makes sense to include it in the email body's default values so that users can easily add it to their email templates. We can include it in the email template this way:
`<p>Email ID: {{ .Values.EmailID }}</p>
`